omod/src/main/java/org/openmrs/module/appointments/web/contract/AppointmentQuery.java

```diff
@@ -12,6 +12,7 @@ public class AppointmentQuery {
     private String patientUuid;
     private String status;
     private String appointmentKind;
+    private Boolean isDatelessAppointments = false;
```

Comment:
renamed to withoutDates

Reply:
"isDatelessAppointments" attribute still exists. Have you refactored the code to search by status instead of relying on "datelessAppointment" as part of search?